### PR TITLE
Feature/tooltip (issue #59)

### LINKED ISSUE #59: Component: Tooltip
Click on text to display additional information.

Let's use https://atomiks.github.io/tippyjs/ for this.

Note the absolute positioning used on the container for the mobile version of the design.

### PR DESCRIPTION
**Pull request by leezaweb**
closes #59

### CODE REVIEW

**Comment by leezaweb** [comment]
Tucker asked for this to be position: fixed

File: assets/_js/spotlights/progress-bar.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const ProgressBar = () => {
   const sh = 'scrollHeight'
   const header = document.querySelector('.site-header')
   const content_height =
-    document.querySelector('.post__content').clientHeight - 100
+    document.querySelector('.spotlight__content').clientHeight - 100
```
Comment (jnschrag) [inline assets/_js/spotlights/progress-bar.js]: Why was this changed?

Comment (jnschrag) [reply assets/_js/spotlights/progress-bar.js]: @leezaweb Tagging so you see this.

Comment (leezaweb) [reply assets/_js/spotlights/progress-bar.js]: @jnschrag  there was a console error because  document.querySelector('.post__content') was null

Comment (leezaweb) [reply assets/_js/spotlights/progress-bar.js]: I changed this on every spotlight component branch btw.
